manipulation/station.py

```diff
@@ -1257,6 +1262,61 @@ def _ApplyDriverConfigsInterface(
         )
 
 
+def _ApplyCameraLcmIdInterface(
+    camera_config,   # See Scenario.cameras for typing
+    camera_id: str, 
+    lcm_buses: LcmBuses,
+    builder: DiagramBuilder,
+) -> None:
+    lcm = lcm_buses.Find("Driver for " + camera_config.name, camera_config.lcm_bus)
+    
+    camera_data_receiver = builder.AddSystem(LcmImageArrayToImages())
+    camera_data_receiver.set_name(camera_config.name + ".data_receiver")
+    camera_data_subscriber = builder.AddSystem(
+        LcmSubscriberSystem.Make(
+            channel=camera_id,
+            lcm_type=lcmt_image_array,
+            lcm=lcm,
+            use_cpp_serializer=True,
+            wait_for_message_on_initialization_timeout=10,
+        )
+    )
+    camera_data_subscriber.set_name(camera_config.name + ".data_subscriber")
+
+    builder.ExportOutput(
+        camera_data_receiver.color_image_output_port(),
+        f"{camera_config.name}.rgb_image",
+    )
+    builder.ExportOutput(
+        camera_data_receiver.depth_image_output_port(),
+        f"{camera_config.name}.depth_image",
+    )
+    builder.ExportOutput(
+        camera_data_receiver.label_image_output_port(),
+        f"{camera_config.name}.label_image",
+    )
+
+    builder.Connect(
+        camera_data_subscriber.get_output_port(),
+        camera_data_receiver.image_array_t_input_port(),
+    )
+
+
+def _ApplyCameraLcmIdsInterface(
+    camera_configs,   # See Scenario.cameras for typing
```

Comment:
why not make it explicit?  or perhaps even remove this method, and just put the for loop in the _MakeHardwareStationInterface method below?